server/api/routes/projects.py

```diff
@@ -1,8 +1,57 @@
 from flask import Blueprint, request
 from database import Projects
-from flask_jwt_extended import jwt_required
+from flask_jwt_extended import get_jwt_identity, jwt_required
 # from database import User
 import json
 
 projects = Blueprint('projects', __name__)
 
+@projects.route('/', methods=['POST'])
+@jwt_required()
+def projects_create():
+    """POST projects/
+    Desc: Creates a new project
+    Returns:
+        201: newly created project
+    """
+    req = request.get_json()
+    req["creator_id"] = get_jwt_identity()["_id"]["$oid"]
+    new_project = Projects(**req).save()
+    return new_project.to_json(), 201
+
+
+@projects.route('/', methods=['GET'])
+@jwt_required()
+def projects_read():
+    """GET projects/
+    Desc: Gets all available projects
+    Returns:
+        200: all projects in the database
+    """
+    return Projects.objects.to_json(), 200
```

Comment:
We should make this so it only returns the projects associated with that specific user. So they can't just look at everyone's projects.